pkg/api/types.go

```diff
@@ -1997,6 +1997,11 @@ func (p ClusterProfile) Secret() string {
 	return fmt.Sprintf("cluster-secrets-%s", name)
 }
 
+// GetClusterProfileSecret returns the default secret name for the profile
+func GetClusterProfileSecret(profile ClusterProfile) string {
```

Comment:
Since this isn't looking at the configuration, it should be named `GetDefaultClusterProfileSecretName` to be more explicit.